install.sh.in

```diff
@@ -58,13 +58,36 @@ nexus-upload raw "${ROOTDIR}/rpms/${RELEASE_NAME}-sle-15sp2" "${RELEASE}-sle-15s
 print_stage "Uploading helm charts"
 nexus-upload helm "${ROOTDIR}/helm" "${CHARTS_REPO:-"charts"}"
 
-print_stage "Deploying loftsman manifests"
-loftsman ship --manifest-path "${ROOTDIR}/manifests/sat.yaml" --charts-path "${ROOTDIR}/helm"
+print_stage "Importing SAT configuration to VCS"
+default_config_import_image_name="registry.local/cray/cf-gitea-import"
+config_import_image=${CONFIG_IMPORT_IMAGE:-$default_config_import_image_name:@CF_GITEA_IMPORT_VERSION@}
+
+# Running the config import will write a file to ${ROOTDIR}/results
+mkdir -p ${ROOTDIR}/results
+
+# Note: CF_IMPORT_GITEA_ORG and CF_IMPORT_CONTENT are set to their defaults.
+podman run --rm --name sat-config-import \
```

Comment:
This could definitely be made into a vendor library function (i.e. make `cf-gitea-import` kind of like how we currently have `cfs-config-util`). I don't think I can get that done in time, but would be a good thing to have with CASM-3025.